kubernetes/run-e2e.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+set -e
+echo -e '- Staring wr Kubernetes End-to-End tests'
+# Environment variables.
+echo -e '- Setting up the test environment..'
+SCRIPT_ROOT=$(dirname ${BASH_SOURCE})/..
+export KUBECONFIG=${KUBECONFIG:-$HOME/.kube/config}
+
+echo ''
+echo -e 'kubernetes version:\t' $(kubectl version -o json | jq .serverVersion.gitVersion)
+echo -e 'wr version:\t' $(git rev-parse --verify HEAD)
+echo ''
+
+# Test the compiled binary as the end user would run it. 
+# Test kubeDeployCmd
+/tmp/wr k8s deploy --debug || (cat ~/.wr_development/kubelog; /bin/false)
```

Comment:
Does this work outside of travis, where I assume you've arranged for /tmp/wr to exist? Make it work locally as well, perhaps by building wr to a temp dir that gets deleted.

Reply:
The makefile puts a copy of wr in /tmp/wr in a step called before this bash script is invoked. I could have an exit behaviour of cleaning up that file quite easily though.